config/deploy/docker/lib/command_args.rb

```diff
@@ -61,6 +62,15 @@ def initialize
 
     raise "Set a valid group: #{ARGV[0]} must be in #{config.keys.join(', ')}" if deployments.nil?
 
+    if ARGV[1] == '--journald'
+      puts 'Using the journald log driver for debugging. Subsequent deployments without this argument will revert back to standard logging'
+      self.log_configuration = {
+        log_driver: 'journald',
+      }
+    else
+      self.log_configuration = :default
+    end
```

Comment:
Is this just for testing, or is there a reason we'd want to revert?

Reply:
I see this as just for testing, meaning while we're debugging the installation with troubles (and not something we'd want to make semi-permanent).